src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -55,12 +55,7 @@ public override void Write<TState>(in LogEntry<TState> logEntry, IExternalScopeP
 
                     if (exception != null)
                     {
-                        string exceptionMessage = exception.ToString();
-                        if (!FormatterOptions.JsonWriterOptions.Indented)
-                        {
-                            exceptionMessage = exceptionMessage.Replace(Environment.NewLine, " ");
-                        }
-                        writer.WriteString(nameof(Exception), exceptionMessage);
+                        writer.WriteString(nameof(Exception), exception.ToString());
```

Comment:
Don't we need to escape the new lines before writing it?

Reply:
Ah, I see it get escaped by `Utf8JsonWriter`. So, this will be fine then. We need to have an E2E test for the case to ensure the newlines are escaped.

Reply:
Exactly, `Utf8JsonWriter` takes care of escaping already, so the change is "trivial".

First I didn't understand the test cases but have fixed them now 👍